### DISCUSSION TITLE
Welcome to Dev-Geeks Discussions!

### DISCUSSION THREAD

**Discussion by pranjay-poddar** [pranjay-poddar/Dev-Geeks#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anushkabh** [comment]
i have today created a pr in hacktoberfest accepted repository ... still its showing not participating . please look into this

**Reply by pranjay-poddar** [reply]
@Anushkabh It is not related to this repository, please check again.

**Reply by pranjay-poddar** [reply]
Dev Geek is a legit repository and it is participating.

**Comment by Anushkabh** [comment]
I don't know why it's showing non participating.. I have created pr in
issue no #444

**Reply by pranjay-poddar** [reply]
It is because you have forked this repo --> created a PR in your own forked repo --> then you have merged the PR by yourself in your repo. 
you have to create a PR in the parent repo ie. pranjay-poddar/Dev-Geeks and not Anushkabh/Dev-Geeks
Hope this helps.

**Comment by Anushkabh** [comment]
Thank you for the clarification.

**Comment by HuyenSuong20222** [comment]
dont trust them :)

**Comment by HuyenSuong20222** [comment]
@Anushkabh see
